docs/reproducibility_resources.md

```diff
@@ -4,40 +4,34 @@ nav_title: Resources
 ---
 
 This list provides some links to resources on reproducibility topics that may be useful as you develop you reproducible research skills and practices.
- 
-Please note, this is not an exhaustive list. 
-It includes multiple types of resources for varying levels of experience, so we would not necessarily recommend every resource here for everyone. 
-Resources are listed by topic and in alphabetical order, not in order of recommendation. 
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+Please note, this is not an exhaustive list.
+It includes multiple types of resources for varying levels of experience, so we would not necessarily recommend every resource here for everyone.
+Resources are listed by topic and in alphabetical order, not in order of recommendation.
 
-### Table of Contents
+**Table of contents**
 
-- [Comprehensive Resources](#comprehensive-resources)
-- [Git](#git)
-- [Project Organization](#project-organization) 
-- [Shell](#shell)
-- [UNIX](#unix)
+* TOC goes here
+{:toc}
 
 ## Comprehensive Resources
 
 + [Bioinformatics Data Skills - Vince Buffalo](https://www.oreilly.com/library/view/bioinformatics-data-skills/9781449367480/)
 + [Collections on Reproducibility - _Nature_](https://www.nature.com/collections/prbfkwmwvz)
-+ [Good Enough Practices in Scientific Computing - _PLOS Computational Biology_](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005510) 
++ [Good Enough Practices in Scientific Computing - _PLOS Computational Biology_](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1005510)
```

Comment:
Makes me think we should cite authors where we can rather than journals. And Doi links?

Reply:
will update all around!